addOns/quickstart/src/main/resources/org/zaproxy/zap/extension/quickstart/resources/Messages.properties

```diff
@@ -12,28 +12,43 @@ quickstart.button.tooltip.launch		= Open the browser you've chosen pre-configure
 quickstart.toolbar.button.tooltip.launch	= Open the browser you've chosen in the Quick Start tab pre-configured to proxy through ZAP
 quickstart.button.label.stop			= Stop
 quickstart.button.tooltip.stop			= Stop the attack
+quickstart.button.label.back			= <
+quickstart.button.tooltip.back			= Back to the main screen
 quickstart.cmdline.outputto				= Writing results to {0}
 quickstart.cmdline.out.help				= The file to write the XML results to
 quickstart.cmdline.progress.help		= Display progress bars while scanning
 quickstart.cmdline.progress.ascan		= Active scanning
 quickstart.cmdline.progress.complete	= Attack complete
 quickstart.cmdline.progress.failed		= Failed to attack the URL
 quickstart.cmdline.progress.notstarted	= Not started
-quickstart.cmdline.progress.spider		= Spidering
+quickstart.cmdline.progress.spider		= Using traditional spider
+quickstart.cmdline.progress.ajaxspider	= Using ajax spider
 quickstart.cmdline.progress.started = Accessing URL
 quickstart.cmdline.progress.stopped		= Manually stopped
 quickstart.cmdline.quickout.error.dirNotWritable = The directory of given ''-quickout'' file is not writable:\n{0}
 quickstart.cmdline.quickout.error.notAFile = The ''-quickout'' value is not a file:\n{0}
 quickstart.cmdline.quickout.error.fileNotWritable = The given ''-quickout'' file is not writable:\n{0}
 quickstart.cmdline.quickout.error.save.report = An error occurred while saving the ''-quickout'' report.
-quickstart.cmdline.quickurl.error.invalidUrl = The provided URL to attack is not valid:
+quickstart.cmdline.quickurl.error.invalidUrl = The provided URL is not valid:
 quickstart.cmdline.quickout.save.report.successful = The report was successfully saved to:\n{0}
 quickstart.cmdline.url.help				= The URL to attack, eg http://www.example.com
 quickstart.field.url.disabled.mode = Attacking arbitrary URLs is not permitted in Protected or Safe mode.
 quickstart.label.show					= Show this tab on start up:
 quickstart.label.explore				= Explore your application:
 quickstart.label.progress				= Progress:
-quickstart.label.url					= URL to attack:
+quickstart.label.attackurl				= URL to attack:
+quickstart.label.ajaxspider				= Use ajax spider:
+quickstart.label.withbrowser			= with
+quickstart.label.exploreurl				= URL to explore:
+quickstart.label.tradspider				= Use traditional spider:
+quickstart.label.hud					= Enable HUD:
+quickstart.link.faq						= Frequently Asked Questions 
+quickstart.link.startguide				= Getting Started Guide
+quickstart.link.usergroup				= User Group 
+quickstart.link.userguide				= Desktop User Guide
+quickstart.link.warning.nostartguide	= Failed to locate or open Getting started guide: {0}
```

Comment:
- Use `\n` instead of space before the parameter? The exception message might be big.
 - `Getting Started Guide`?